src/python/pants/engine/legacy/graph.py

```diff
@@ -361,7 +361,8 @@ def hydrate_target(target_adaptor):
                         tuple(target_adaptor.dependencies))
 
 
-def _eager_fileset_with_spec(spec_path, filespec, snapshot, include_dirs=False):
+# This is deprecated and should not be used in new code.
```

Comment:
Did you rename this for the purposes of the unit test? Should be able to leave it private and test it if so.

Reply:
Undone